cluster/addons/calico-policy-controller/README.md

```diff
@@ -3,13 +3,6 @@
 
 Calico is an implementation of the Kubernetes network policy API.  The provided manifest installs a DaemonSet which runs Calico on each node in the cluster.
 
-### Templating
```

Comment:
Instead of removing this directly, shall we clarify that the calico policy will use ip-masq-agent for MASQUERADE here?